test/param/int-list.spec.js

```diff
@@ -0,0 +1,134 @@
+const expect = require('chai').expect;
+const { describe } = require('node-tdd');
+const api = require('../../src/index').Api();
+
+describe('Testing IntList Parameter', () => {
+  let queryParam;
+  let jsonParam;
+  before(() => {
+    queryParam = api.IntList('list', 'query');
+    jsonParam = api.IntList('list', 'json');
+  });
+
+  it('Testing valid query parameter', () => {
+    expect(queryParam.get({
+      queryStringParameters: {
+        list: '[10, 20]'
+      }
+    })).to.deep.equal([10, 20]);
+  });
+
+  it('Testing invalid query parameter', () => {
+    expect(() => queryParam.get({
+      queryStringParameters: {
+        list: 'invalid'
+      }
+    })).to.throw('Invalid Value for query-Parameter "list" provided.');
+  });
+
+  it('Testing invalid float number query parameter', () => {
+    expect(() => queryParam.get({
+      queryStringParameters: {
+        list: '[10.1, 20]'
+      }
+    })).to.throw('Invalid Value for query-Parameter "list" provided.');
+  });
+
+  it('Testing valid json parameter', () => {
+    expect(jsonParam.get({
+      body: {
+        list: [123, 345]
+      }
+    })).to.deep.equal([123, 345]);
+  });
+
+  it('Testing invalid json parameter', () => {
+    expect(() => jsonParam.get({
+      body: {
+        list: ['123', 213]
+      }
+    })).to.throw('Invalid Value for json-Parameter "list" provided.');
+  });
+
+  describe('Testing optional int parameter "minItemValue"', () => {
+    let queryParamInValidMin;
+    let bodyParamInValidMin;
+    before(() => {
+      queryParamInValidMin = api.IntList('list', 'query', { minItemValue: 1 });
+      bodyParamInValidMin = api.IntList('list', 'json', { minItemValue: 1 });
+    });
+
+    it('Testing invalid min-value of list item in query param', () => {
+      expect(() => queryParamInValidMin.get({
+        queryStringParameters: {
+          list: '[0, 1.2]'
```

Comment:
```suggestion
          list: '[0, 1]'
```

Reply:
I'm going to let it as `list: '[0, 2]'` for consistency with JSON param test that is using same params